.github/workflows/ci.yml

```diff
@@ -3,7 +3,27 @@ name: Run unit tests & check conventions
 on: [push, pull_request]
 
 jobs:
-  tests:
+  file-conventions:
+    runs-on: ubuntu-22.04
+    container:
+      image: "ubuntu:22.04"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install required dependencies
+        run: |
+          apt update
+          apt install -y sudo
+          sudo apt install -y curl
+          sudo apt-get install -y libicu-dev
```

Comment:
I don't think these 2 lines above is needed anymore ^

Reply:
removing libicu-dev causes no problems but when I remove curl CI fails

Reply:
> sure, leave curl there then (BTW can you point me URL of the CI that failed when curl is not there?, just curious);

Sure: https://github.com/nblockchain/conventions/actions/runs/3486959767/jobs/5834031646

Reply:
I doubt that's caused by missing curl, please re-run that CI

Reply:
I didn't find the re-run button in the previous jobs so I removed curl again and re-committed. It failed again:
https://github.com/nblockchain/conventions/actions/runs/3487648476/jobs/5835523129

This time there was a re-run jobs button so I re-ran the failing job and failed again.

Reply:
You can see in the last two commits in the commit history

Reply:
that is really weird; anyway let's leave curl and move on, but add a comment explaining why

Reply:
Done